test/Common/ServerFixture.cs

```diff
@@ -35,30 +34,20 @@ public ServerFixture()
             StartServer();
         }
 
-        public class Startup
+        private void StartServer()
         {
-            public void ConfigureServices(IServiceCollection services)
-            {
-                services.AddSockets();
-                services.AddSignalR();
-                services.AddEndPoint<EchoEndPoint>();
-            }
-
-            public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+            var startupType = GetType().Assembly.GetTypes().SingleOrDefault(type => type.Name == "Startup");
```

Comment:
StartServer is being called from the ctor and ServerFixture is instantiated by XUnit because we use ICollectionFixture.